cyphort.json

```diff
@@ -0,0 +1,826 @@
+{
+    "appid": "ac4f5ee5-2856-4863-b64e-d669727f44af",
+    "name": "Cyphort",
+    "description": "This app supports executing investigative actions to analyze executables on the Cyphort sandbox",
+    "type": "sandbox",
+    "main_module": "cyphort_connector.py",
+    "app_version": "2.0.1",
+    "utctime_updated": "2021-12-02T06:49:07.000000Z",
+    "package_name": "phantom_cyphort",
+    "product_vendor": "Cyphort",
+    "product_name": "Cyphort",
+    "product_version_regex": ".*",
+    "min_phantom_version": "5.0.0",
+    "publisher": "Splunk Community",
+    "python_version": "2.7",
```

Comment:
I assume the python version is okay given that it is a community app?

Reply:
Correct Tony but this app was previously certified and due to lack of assets, we are converting it as a community app. we have added a python version to avoid python version conflict and contributors can convert it to py3